### PR TITLE
Fix fleet for OTEL in Kubewarden 1.20

### PR DESCRIPTION
**Pull request by kravciak**
Update fleet example from https://github.com/kubewarden/docs/pull/476

### CODE REVIEW

**Review by jvanz — APPROVED** [review]

**Comment by flavio** [comment]
Should we start tagging this repository to make it easier to understand how the configuration evolves over the time?

For example, the current `HEAD` should be tagged `v1.19.0`, while this one should go under the `v1.20.0`.

I see one disadvantage of that: most of the times this configuration doesn't change when there's a new release of Kubewarden. That would be one more thing to keep in mind when doing releases, plus we would have **a lot** of tags pointing to the same git commit

Another option would be to simply add a comment into the README, stating which version of KW the configuration has been tested against.

What do the other @kubewarden/kubewarden-developers think about that, do you have other ideas?

**Comment by flavio** [comment]
Moving to the blocked column, we will merge this once 1.20 is out.

Moreover, before merging this one, can you please add a note to the README stating this was tested with KW 1.20.0? Thanks!

**Comment by kravciak** [comment]
Waiting for renovate bot to bump kubewarden to 1.20

**Review by viccuad — APPROVED** [review]

**Pull request merged**
